CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.19.1] - 2021-01-14
```

Comment:
You forgot to update the version in the changelog :)

Reply:
Je me disais aussi que c'était trop simple...